modules/developers-guide/pages/tutorials/phonebook.adoc

```diff
@@ -72,14 +71,11 @@ Creating user-defined types improves the readability of the code.
 * The `+Phone+` type is identified as an optional value by using the `+?Phone+` syntax.
 --
 
-== Start the local network
+== Start the local canister execution environment
 
-Before you can build the `+phonebook+` project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+For development purposes `dfx` provides a local canister execution environment. This requires a `+dfx.json+` file, so you should be sure you are in your project's root directory. For this tutorial, you should have two separate terminal shells, so that you can start and see the output of the processes related to the local canister execution environment in one terminal and manage your project in another.
```

Comment:
Agreed.
```suggestion
For development purposes `dfx` provides a local canister execution environment. This requires a `+dfx.json+` file, so you should be sure you are in your project's root directory. For this tutorial, you should have two separate terminal shells, so that you can start and see the output of the local canister execution environment in one terminal and manage your project in another.
```